src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      data-qa="big"
+      method="post"
+    >
+      <div class="search-field search-field--big">
+        <button class="search-field__button">
+          <img
+            src="./images/Search.svg"
+            alt="Search"
+            class="search-field__icon"
+            type="submit"
```

Comment:
it doesn't have type submit

Reply:
It was mistake.
`type="submit"` was moved to the `button`.